core/server/master/src/test/java/alluxio/master/file/FileSystemMasterSyncMetadataMetricsTest.java

```diff
@@ -0,0 +1,506 @@
+  @Rule
+  public TemporaryFolder mTempDir = new TemporaryFolder();
+  private String mUfsUri;
+  private ExceptionThrowingLocalUnderFileSystem mUfs;
+  private ExecutorService mExecutorService;
+  private MasterRegistry mRegistry;
+  private DefaultFileSystemMaster mFileSystemMaster;
+
+  @Before
+  public void before() throws Exception {
+    UserState s = UserState.Factory.create(ServerConfiguration.global());
+    AuthenticatedClientUser.set(s.getUser().getName());
+
+    mTempDir.create();
+    mUfsUri = mTempDir.newFolder().getAbsolutePath();
+
+    mUfs = new ExceptionThrowingLocalUnderFileSystem(new AlluxioURI(mUfsUri),
+        UnderFileSystemConfiguration.defaults(ServerConfiguration.global()));
+    PowerMockito.mockStatic(UnderFileSystem.Factory.class);
+    Mockito.when(UnderFileSystem.Factory.create(anyString(), any())).thenReturn(mUfs);
+
+    ServerConfiguration.set(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.UFS);
+    ServerConfiguration.set(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, mUfsUri);
+    String journalFolderUri = mTempDir.newFolder().getAbsolutePath();
+
+    mExecutorService = Executors
+        .newSingleThreadExecutor(ThreadFactoryUtils.build("DefaultFileSystemMasterTest-%d", true));
+    mRegistry = new MasterRegistry();
+    JournalSystem journalSystem =
+        JournalTestUtils.createJournalSystem(journalFolderUri);
+    CoreMasterContext context = MasterTestUtils.testMasterContext(journalSystem);
+    new MetricsMasterFactory().create(mRegistry, context);
+    BlockMaster blockMaster = new BlockMasterFactory().create(mRegistry, context);
+    mFileSystemMaster = new DefaultFileSystemMaster(blockMaster, context,
+        ExecutorServiceFactories.constantExecutorServiceFactory(mExecutorService));
+    mRegistry.add(FileSystemMaster.class, mFileSystemMaster);
+    journalSystem.start();
+    journalSystem.gainPrimacy();
+    mRegistry.start(true);
+
+    MetricsSystem.resetAllMetrics();
+  }
+
+  @After
+  public void after() throws Exception {
+    mRegistry.stop();
+    mExecutorService.shutdown();
+  }
+
+  @Test
+  public void testMetadataSyncMetrics() throws Exception {
+    final Counter count = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_COUNT;
+    final Counter success = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_SUCCESS;
+    final Counter fail = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_FAIL;
+    final Counter noChange = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_NO_CHANGE;
+    final Counter skipped = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_SKIPPED;
+    final Counter pathSuccess =
+        DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_SYNC_PATHS_SUCCESS;
+    final Counter pathFail = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_SYNC_PATHS_FAIL;
+    final Counter timeMS = DefaultFileSystemMaster.Metrics.INODE_SYNC_STREAM_TIME_MS;
+
+    createDir("/dir0");
+    OutputStream outputStream = createFile("/dir0/file0");
```

Comment:
Define a `private static final TEST_FILE = "/dir0/file0"` in the test class and use it in your tests

Reply:
done